src/olympia/addons/templates/addons/impala/developers.html

```diff
@@ -99,9 +99,9 @@ <h1 class="addon"{{ addon.name|locale_html }}>{{ title }}</h1>
           {{ author|user_vcard }}
         </div>
         <div class="prose">
-          {% if author.bio %}
+          {% if author.biography %}
             <h3>{{ _('Who is {0}?')|f(author.name) }}</h3>
-            <p class="bio">{{ author.bio|nl2br }}</p>
+            <p class="biography">{{ author.biography|nl2br }}</p>
```

Comment:
The `|no_links` should be redundant (in the other template too) as we prevent links specifically during profile edit, but yeah.